src/features/user/TreeMapper/Analytics/components/SpeciesPlanted/index.tsx

```diff
@@ -0,0 +1,269 @@
+            stops: [0, 100],
+            opacityFrom: 0.4,
+            opacityTo: 0.5,
+          },
+        },
+      },
+      tickPlacement: 'on',
+    },
+    yaxis: {
+      logarithmic: true, //open bug that causes data labels to render wrong numbers
+      axisBorder: {
+        show: true,
+      },
+      labels: {
+        show: true,
+        formatter: function (val) {
+          return getFormattedNumber(language, val);
+        },
+      },
+    },
+  });
+
+  useEffect(() => {
+    if (project) {
+      const FILE_NAME = `${project?.name}__${t('speciesPlanted')}__${format(
+        fromDate,
+        'dd-MMM-yy'
+      )}__${format(toDate, 'dd-MMM-yy')}`;
+
+      setOptions({
+        ...options,
+        chart: {
+          ...options.chart,
+          toolbar: {
+            ...options!.chart!.toolbar,
+            export: {
+              ...options!.chart!.toolbar!.export,
+              csv: {
+                ...options!.chart!.toolbar!.export!.csv,
+                filename: FILE_NAME,
+              },
+              svg: {
+                filename: FILE_NAME,
+              },
+              png: {
+                filename: FILE_NAME,
+              },
+            },
+          },
+        },
+      });
+    }
+  }, [project, toDate, fromDate]);
+
+  const getPlotingData = (speciesData: Species[]) => {
+    const speciesPlanted: number[] = [];
+    const categories: string[] = [];
+
+    for (const species of speciesData) {
+      speciesPlanted.push(species.total_tree_count);
+      categories.push(species.name as string);
+    }
+
+    return { speciesPlanted, categories };
+  };
+
+  const fetchPlantedSpecies = async () => {
+    const res = await fetch('/api/analytics/species-planted', {
+      method: 'POST',
+      body: JSON.stringify({
+        projectId: project!.id,
+        startDate: fromDate,
+        endDate: toDate,
+      }),
+    });
+    const { data }: { data: Species[] } = await res.json();
+
+    let unknownIndex = -1;
+
+    const speciesData = data.map((species, index) => {
+      if (species.other_species === 'Unknown') {
+        unknownIndex = index;
+      }
+      if (!species.name && species.other_species !== 'Unknown') {
+        return { ...species, name: species.other_species };
+      }
+      return species;
+    });
+
+    if (unknownIndex !== -1) {
+      const unknownSpecies = speciesData.splice(unknownIndex, 1)[0];
+      speciesData.push({
+        ...unknownSpecies,
+        name: unknownSpecies.other_species,
+      });
+    }
+
+    const { speciesPlanted, categories } = getPlotingData(speciesData);
+
+    console.log('==>', speciesPlanted, categories);
+
+    setSeries([
+      {
+        data: speciesPlanted,
+        name: t('speciesPlanted'),
+      },
+    ]);
+
+    setOptions({
+      ...options,
+      xaxis: { ...options.xaxis, categories: categories },
+    });
+  };
+
+  useEffect(() => {
+    if (project) {
+      fetchPlantedSpecies();
+    }
+  }, [project]);
```

Comment:
Make this depend from the fromDate and toDate as well?
```suggestion
  }, [project, fromDate, toDate]);
```